.github/workflows/build.yml

```diff
@@ -27,11 +27,33 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     outputs:
+      # NOTE: Some of the referenced steps set outputs conditionally and there
```

Comment:
Please could you delete all these NOTE prefixes? All comments are things to note :) 

```suggestion
      # Some of the referenced steps set outputs conditionally and there
```

Also, something I missed in https://github.com/python/cpython/pull/121766, please could you replace the ~forbidden "no-GIL" with "free-threaded"? We can do it in this PR.

Reply:
@hugovk could you point to where I kept the no-GIL reference? It's not visible in this diff. I remember having to use it to keep the commit messages under limits or smth like that, but I can't seem to find it anymore.

Reply:
Oh, it's in the inputs. Okay, will do.

Reply:
@hugovk done, and I'm marking this thread as resolved.